.github/workflows/security-alerts.yml

```
@@ -0,0 +1,59 @@
+name: Security Advisory Alert
```

```suggestion
name: DataHub Security Advisory Alert
```
these security advisories are from DataHub right?